public/pages/CreateMonitor/containers/DefineMonitor/utils/mappings.test.js

```diff
@@ -20,18 +20,16 @@ describe('getPathsPerDataType', () => {
     const mappings = {
       random_index: {
         mappings: {
-          _doc: {
-            properties: {
-              '@message': { type: 'text' },
-              '@timestamp': { type: 'date' },
-              username: { type: 'keyword' },
-              memory: { type: 'double' },
-              phpmemory: { type: 'long' },
-              bytes: { type: 'long' },
-              clientip: { type: 'ip' },
-              id: { type: 'integer' },
-              ip: { type: 'ip' },
-            },
+          properties: {
```

Comment:
Was the removal of types (_doc) a breaking change or a deprecation in 7.0? If it's just a deprecation we'd have to expect both.

Reply:
in 7.0 it is deprecated. but this use case is around using [get mapping](https://www.elastic.co/guide/en/elasticsearch/reference/7.0/indices-get-mapping.html) which will always return in the format we're expecting unless explicitly setting value.